nengo_gui/page.py

```diff
@@ -238,6 +239,10 @@ def execute(self, code):
                 self.error = dict(trace='must declare a nengo.Network '
                                   'called "model"', line=line)
             model = None
+        else:
+            # set all the seeds so that creating components doesn't affect
+            #  the neural model itself
+            nengo_gui.seed_generation.define_all_seeds(model)
```

Comment:
I think this is not the right place to do the seed generation:
* It does not take effect for all possible ways to instantiate the GUI. For example, try `IPythonViz` with a top level seed and a single ensemble with a single neuron. For me it produces two different tuning curves depending on whether I insert a value plot or not.
* It happens for every change in the editor which might be expensive.

I think it might be better to do this just before the build happens.